py-polars/polars/expr/expr.py

```diff
@@ -6469,6 +6487,7 @@ def rolling_std(
         by: str | None = None,
         closed: ClosedInterval = "left",
         ddof: int = 1,
+        warn_if_unsorted: bool = True,
```

Comment:
I think we should flag this argument as experimental? I think the goal is to remove this argument and ensure sortedness is preserved.

Reply:
Sure, done

Even if it's done on the user's behalf, I think @orlp wanted to give a performance warning if sorting is done on the user's behalf https://github.com/pola-rs/polars/issues/10991#issuecomment-1712042866